src/reducers/sources.js

```diff
@@ -33,6 +33,10 @@ export type SourcesState = {
     line?: number,
     column?: number
   },
+  highlightedLineRange?: {
+    start?: number,
+    end?: number
+  },
```

Comment:
I think this would be nicer in the UI reducer as it has more to do with some UI state. 

Also, we should probably include a sourceID so that we can automatically clear this if the user navigates to another source.